### DISCUSSION TITLE
How come you didn't use a template langue instead of inventing your own?

### DISCUSSION THREAD

**Discussion by AnderssonPeter** [atifaziz/Transplator#10] [Q&A]
How come you didn't use a template langue instead of inventing your own?
There are alot of nugets out there that have template languages like Fluid, Cottle, Scriban and so on.

**Comment by atifaziz** [comment]
There is no templating language being invented. The templating language, if you will, is really just C#, and then you have tokens like `{%` and `%}` used to delimit sections of C# code from literal text.
